evap/evaluation/tests/test_tools.py

```diff
@@ -50,10 +50,13 @@ def test_log_exceptions_decorator(self, mock_logger, __):
             One could create a mock management command and call its handle method manually,
             but to me it seemed safer to use a real one.
         """
+        exception_was_reraised = False
         try:
             management.call_command('update_evaluation_states')
         except Exception:  # pylint: disable=broad-except
-            pass
+            exception_was_reraised = True
```

Comment:
As I understand this test, you transform `Evaluation.update_evaluations()` into a saboteur which raises a generic `Exception`.
What do you think about using a specific `class SaboteurException(Exception)` subclass for the saboteur and using `assertRaises`?

Reply:
fancy idea, I like it :+1: